src/Models/ModelFactory.php

```diff
@@ -170,7 +191,7 @@ private static function resolveListColumns(stdClass $response): stdClass
     private static function resolveLocaleValues(string $locale, mixed $input)
     {
         if (is_object($input)) {
-            foreach ($input as $key => &$value) {
```

Comment:
@rikzwarthoff deze change geeft problemen! Doordat de oorspronkelijke `$responseObject` wordt aangepast (doordat die hier *referenced* is), is het onmogelijk om de oorspronkelijke vertalingen te scoren van bijv. `label`. Deze label incl. vertalingen wordt dan altijd overschreven met puur de waarde van de default language. Hierdoor kan ik de verschillende vertalingen nooit meer benutten...

In deze change heb ik die referenced weggehaald, maar dan werkt de rest van de logica niet meer. Bijv. bij een product sync:
`php artisan product:sync 1 APPLE_MVVK2_32GB_2TB_QWERTY_US`
Exception:
`Object of class stdClass could not be converted to string
  at vendor/flooris/ergonode-api-laravel/src/Models/ModelFactory.php:138`